### PR TITLE
Feedback from Scott

### PR DESCRIPTION
**Pull request by codeBelt**

### CODE REVIEW

**Comment by scottbromander** [comment]
Eww.. 

```
export enum HttpVerbs {
  Delete = 'DELETE',
  Get = 'GET',
  Patch = 'PATCH',
  Post = 'POST',
  Put = 'PUT',
}
```

Haha!

File: src/components/pages/index-page/IndexPage.store.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    *loadShow() {
+      const response: ApiResponse<IShow> = yield getShowRequest(defaultShowId);
+
+      this.showResults = {
+        data: this.showResults.data,
+        isRequesting: false,
+        ...response, // Overwrites the default data prop or adds an error. Also adds the statusCode.
+      };
+    },
```
Comment (codeBelt) [inline src/components/pages/index-page/IndexPage.store.ts:43]: @scottbromander

> One thing really stands out to me as odd. You use the show ID in the url params of the episode list, but not the index. 

I was thinking of having a dropdown with id's on the homepage to set the show id but I didn't want to do more than I needed for the example.

File: src/components/pages/about-page/AboutPage.store.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      ...response, // Overwrites the default data prop or adds an error. Also adds the statusCode.
+    };
+
+    if (response.error) {
+      const message = `${response.statusCode}: ${response.error.message}`;
+
+      this.globalStore.toastStore.enqueueToast(message, 'error');
+    }
+  }
+}
```
Comment (codeBelt) [inline src/components/pages/about-page/AboutPage.store.ts:37]: @scottbromander from your experience in working with JavaScript developers do they not use Classes that often? I have found that is true for me and that is why I find myself working less and less with Classes. The other stores are just functions.

File: src/components/pages/episodes-page/episodes-toggle/EpisodesToggle.constants.tsx [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export enum EpisodesToggleOption {
+  ASC = 'asc',
+  DESC = 'desc',
+}
```
Comment (codeBelt) [inline src/components/pages/episodes-page/episodes-toggle/EpisodesToggle.constants.tsx:4]: @scottbromander  You said you don't like. How would you like to see it?

```ts
export enum HttpVerbs {
  Delete = 'DELETE',
  Get = 'GET',
  Patch = 'PATCH',
  Post = 'POST',
  Put = 'PUT',
}
```

File: src/components/pages/index-page/IndexPage.store.ts (hunk 1 above)
Comment (codeBelt) [reply src/components/pages/index-page/IndexPage.store.ts:43]: @scottbromander what would you do?

File: src/components/pages/episodes-page/episodes-toggle/EpisodesToggle.constants.tsx (hunk 3 above)
Comment (codeBelt) [reply src/components/pages/episodes-page/episodes-toggle/EpisodesToggle.constants.tsx:4]: @scottbromander sometimes you just have to follow what the standard is already `PascalCase`:

https://www.typescriptlang.org/docs/handbook/enums.html

Sure I could see ALL_CAPS for single constant variables but not for enums.

**Pull request closed without merging**